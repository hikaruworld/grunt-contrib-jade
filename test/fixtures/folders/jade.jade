#test.test
  span#data data