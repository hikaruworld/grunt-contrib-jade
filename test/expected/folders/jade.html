<div id="test" class="test"><span id="data">data</span></div>